### DISCUSSION TITLE
Acharya 0.3.4-alpha release

### DISCUSSION THREAD

**Discussion by vimal-m** [astutic/Acharya#11] [General]
# Release notes - Acharya Trainer - 0.3.4-alpha

### Bugs Fixed

[AT-565] In training output, under per record evaluation scores, show the correct and wrong entry details in UI

[AT-568] In workbench, highlight mode, disable mouse hover for grayed out tags

[AT-578] Dictionary widget: Read more URL is incorrect

### Tasks Completed

[AT-213] Sort evaluated records based on accuracy score

[AT-542] Create UI for data export

[AT-564] In workbench have an option to show the Algo evaluated record side-by-side the annotated record

[AT-573] Add links to the record numbers in accuracy per record in training reports to open the records in workbench

[AT-576] Ability to pass parameters to Algo docker container

[AT-579] Upload UI is confusing for first time users. Make it more user friendly

[AT-580] In upload screen modify the form layout to make it easier to understand the screen

[AT-581] Remove coreui dependency

[AT-582] Remove ngx-monacoeditor dependency

<hr /><em>This discussion was created from the release <a href='https://github.com/astutic/Acharya/releases/tag/v0.3.4-alpha'>Acharya 0.3.4-alpha release</a>.</em>

**Comment by ameyDH** [comment]
Windows FIrewall prevent it from installing and treat it as a malware

**Reply by vimal-m** [reply]
Acharya is a terminal-based product and logs are printed onto the terminal output. Please run the exe via cmd-prompt/terminal/powershell. 
Acharya as it is a stand-alone exe, it does not install anything on the system. 
Acharya service listens on port 3000 by default when it runs (to enable access to its web-UI), and the windows firewall would ask for access to port 3000 as shown in the screenshot in this doc https://acharya.astutic.com/docs/get-started/acharya-installation#for-windows

When you double click the exe to run via the explorer, Windows firewall complaints about this because the publisher of the exe is Unknown. Since Acharya is a community edition product, getting a Windows code signing certificate would be an added cost to us.
